docs/website/docs/development/devfile.md

```diff
@@ -82,14 +82,64 @@ The order in which the commands are ran for `odo dev` are:
 3. `run`: The application is ran within the container
 4. `debug`: This is ran when `odo dev --debug` is executed
 
-When `odo deploy` is executed, we use: deploy.
+When `odo deploy` is executed, we use: `deploy`.
 
 These commands are typically tied to Kubernetes or OpenShift inline resources. They are defined as a component. However, you can use `container` or `image` components as well under the deploy group.
 
 The most common deploy scenario is the following:
 1. Use the `image` component to build a container
 2. Deploy a `kubernetes` component(s) with a Kubernetes resource defined
 
+### How odo runs exec commands in Deploy mode
+```yaml
+commands:
+  - exec:
+      commandLine: helm repo add bitnami https://charts.bitnami.com/bitnami && helm install my-db bitnami/postgresql
```

Comment:
nit: just to avoid scrolling horizontally to see the command:

```suggestion
      commandLine: |
        helm repo add bitnami https://charts.bitnami.com/bitnami && \
        helm install my-db bitnami/postgresql
```